.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  build_zips:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - otp: 22
+            otp_version: 22.3
+          - otp: 23
+            otp_version: 23.3
+          - otp: 24
+            otp_version: 24.1
+            build_docs: build_docs
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ matrix.otp_version }}
+          version-type: strict
+      - name: Build Elixir Release
+        run: |
+          vsn=$(cat VERSION)
+          make Precompiled.zip
+          mv Precompiled-v${vsn}.zip elixir-otp-${{ matrix.otp }}.zip
+          echo "$PWD/bin" >> $GITHUB_PATH
+      - uses: actions/upload-artifact@v2
+        with:
+          name: elixir-otp-${{ matrix.otp }}.zip
+          path: elixir-otp-${{ matrix.otp }}.zip
+      - uses: actions/checkout@v2
+        with:
+          repository: elixir-lang/ex_doc
+          path: ex_doc
+        if: ${{ matrix.build_docs }}
+      - name: Build ex_doc
+        if: ${{ matrix.build_docs }}
+        run: |
+          mv ex_doc ../ex_doc
+          cd ../ex_doc
+          ../elixir/bin/mix do local.rebar --force, local.hex --force, deps.get, compile
+          cd ../elixir
+      - name: Build Docs
+        if: ${{ matrix.build_docs }}
+        run: |
+          vsn=$(cat VERSION)
+          make Docs.zip
+          mv Docs-v${vsn}.zip Docs.zip
+      - uses: actions/upload-artifact@v2
+        with:
+          name: Docs.zip
+          path: Docs.zip
+        if: ${{ matrix.build_docs }}
+  create_draft_release:
+    needs: build_zips
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-22.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-23.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-24.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: Docs.zip
+      - name: Rename files
+        run: |
+          vsn=$(cat VERSION)
+          mv elixir-otp-22.zip elixir-${vsn}-otp-22.zip
+          mv elixir-otp-23.zip elixir-${vsn}-otp-23.zip
+          mv elixir-otp-24.zip elixir-${vsn}-otp-24.zip
```

Comment:
```suggestion
```

I don't think we need this step. The Elixir version will already be in the Release tag. That's enough, no?

Reply:
My intention is to rely on VERSION file instead of github.ref_name. But i can use the ref_name instead for this case. What do you think?

Reply:
Right now it doesn’t matter because they will always be the same but I think in the future the ref name is better when we do nightly builds because we may have a short period of time where the version in the main branch is v1.14.0 while there is also a v1.14.0 tag.

Reply:
But the main point is that we don’t need those in the archive names I think. :)